packages/aws-cdk-lib/aws-cloudfront/lib/distribution.ts

```diff
@@ -129,7 +129,7 @@ export interface DistributionProps {
    *
    * If you want to use your own domain name, such as www.example.com, instead of the cloudfront.net domain name,
    * you can add an alternate domain name to your distribution. If you attach a certificate to the distribution,
-   * you must add (at least one of) the domain names of the certificate to this list.
```

Comment:
Since this is no longer something required, could you also add a sentence at the end explaining the case where we don't need to provide a domain name?